.devfile

```diff
@@ -0,0 +1,14 @@
+specVersion: 0.0.1
+name: che-in-che
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
```

Comment:
Can you elaborate on why types differ from types in Che plugin registry?

Reply:
I think the answer for now - because it was originally defined like that in the reference implementation.  https://github.com/redhat-developer/devfile/blob/master/petclinic.yaml#L25
Do you want to initiate a task to syncronize this names?